middle_end/flambda2/simplify/inlining/inlining_report.mli

```diff
@@ -31,9 +31,12 @@ type at_call_site =
     simplifying the body, and one after (this is useful for e.g. recursive
     functions). *)
 type fundecl_pass =
+  | After_closure_conversion
   | Before_simplify
   | After_simplify
```

Comment:
you need to add a `(**)` before the actual doc comment, else the doc comment is attached to the last constructor and not the whole type declaration

Reply:
Looks like there's already a doc comment for the type declaration. Probably an oversight?